orchestra/static/orchestra/common/components/checklist/partials/checklist.html

```diff
@@ -0,0 +1,21 @@
+<div class="checklist-wrapper">
+    <div ng-if="data.items.length" class="toggle-completed"
+         ng-click="data.hideCompleted = !data.hideCompleted">
```

Comment:
It's undefined on init, so essentially false, but explicit is better than implicit.